app/index.js

```diff
@@ -200,29 +205,58 @@ var NemoGenerator = yeoman.generators.Base.extend({
       var newval = "<%=" + val + "%>";
       return newval;
     };
-    this.template('_loopmocha.js', taskDir + 'loopmocha.js');
+    if(this.testFramework==='mocha'){
```

Comment:
Missing spaces around ===. I see this in a few places. Obviously stylistic only but pretty standard. If we were using jshint as a part of development, it would complain.. I'll file an issue that jshint should be a part of the build process going forward.